GUI.py

```diff
@@ -455,11 +499,14 @@ def init_ui(self):
         self.result_layout = QVBoxLayout()
         self.oper_lbl = QLabel(self.w)
         self.pretest_lbl = QLabel(self.w)
+        self.scroll = QScrollArea(self.w)
+
+        self.scroll.setVerticalScrollBarPolicy(Qt.ScrollBarAlwaysOn)
+        self.scroll.setWidgetResizable(True)
+        self.scroll.resize(900, 600)

```

Comment:
Does something in here limit the size? It doesn't scroll enough to show all of the ordinal results

Reply:
Fixed by resetting the window